assets/src/styles/blocks/Columns.scss

```diff
@@ -4,32 +4,28 @@
       font-weight: inherit;
     }
 
-    &.btn-primary {
```

Comment:
In the default theme this is the same value as other primary buttons, so it seems this shouldn't be needed? If we don't remove this rule then all primary buttons variables are applied except the text always stays white.

Reply:
Another option is to make this one variable too, but in that case it might be better to give it its own button class instead of making adjustments to the primary button?